src/test/.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": ["airbnb-base", "plugin:jest/recommended"],
+  "env": {
+    "jest": true
+  },
+  "plugins": ["jest"],
+  "parser": "babel-eslint",
+  "rules": {
+    "indent": [2, 4],
+    "no-unused-vars": 0,
```

Comment:
i would prefer enforcing certain styles and fixing the current errors instead of turning of the rules. E.g. the comma-dangle rule could be configured to enforce not using commas.

For certain rules like no-underscore-dangle we could have exceptions per next-line?